### ISSUE TITLE
Improve player detection

### ISSUE THREAD

**Issue by McCavity** [McCavity/Behringer_CMD_MM-1#2]
In procedures:

OnPlayerStart
OnPlayerStop
OnPlayerEOFWarning
OnPlayerStateChange
OnPlayerPFLOn
OnPlayerPFLOff

```
see if the logic can be simplified by using a direct player index (like in OnLoad)
```

**Comment by McCavity** [comment]
The method of identifying players in OnLoad is much more elegant than the current solution in the listed procedures. Check and try if this can be improved

**Comment by McCavity** [comment]
merged into master with #9
